### ISSUE TITLE
Low quality pokemon sprites on windows

### ISSUE THREAD

**Issue by Jupiee** [andrewbenington/OpenHome#89]
I built the recent commit and tested on windows, opened a Gen 6 (Omega Ruby) main file and it shows low quality pokemon images on windows. I didn't test it for other games but I am pretty sure it's the same result since its using the same spritesheet for box icons

![image](https://github.com/user-attachments/assets/cc8b68c8-ce36-4c5a-b7a1-11d0aff46b1f)

**Comment by andrewbenington** [comment]
This is expected, Gen 6 PKMs use sprites from in-game models so the pixelation is due to the lower resolution of the 3DS. If you select "OpenHome" from the top right dropdown, it will show the sprite from Pokémon Home, which is higher quality. Unless I'm missing the issue?

**Comment by andrewbenington** [comment]
This is the source of that particular sprite: https://bulbapedia.bulbagarden.net/wiki/Camerupt_(Pok%C3%A9mon)#/media/File:Spr_6x_323_m.png

**Comment by Jupiee** [comment]
> This is expected, Gen 6 PKMs are displayed using Gen 6 3DS models. The pixelation is due to the lower resolution of the 3DS. If you select "OpenHome" from the top left dropdown, it will show the sprite from Pokémon Home, which is higher quality. Unless I'm misunderstanding the issue?

ok this makes more sense, OpenHome sprites look much better. Why not keep OpenHome sprites default? because some of these gen 6 sprites look really bad like this one

Gen 6:
![image](https://github.com/user-attachments/assets/3af5a000-921a-4985-968e-fbbf913e6914)
OpenHome:
![image](https://github.com/user-attachments/assets/2b8f0f8d-6f1c-4d46-bf2a-8ae090c0b928)

**Comment by andrewbenington** [comment]
Yeah I get that, maybe it could be a toggle on the setting page? Where the format-specific sprites can be disabled for each format

**Comment by evanwporter** [comment]
>Where the format-specific sprites can be disabled for each format

Isn't this just a resource pack

**Comment by andrewbenington** [comment]
> > Where the format-specific sprites can be disabled for each format
> 
> Isn't this just a resource pack

yep that's what this turned into (#95)
